drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -216,6 +216,12 @@ def dob
       object.dob if current_permission?(permission: :can_view_dob, entity: object)
     end
 
+    def first_name
+      return 'Anonymous' if object.first_name.blank? && object.last_name.blank?
```

Comment:
Is this a request from someone?  Historically we've named these something like `"Anonymous #{object.personal_ID}"` to prevent everyone getting the same name and then being auto-merged or impossible to distinguish.

Reply:
Removing this, I'm going to move it to the frontend-only to address this and theron's concern https://github.com/greenriver/hmis-warehouse/pull/3534#discussion_r1332259735

Reply:
And no it wasn't a request, it just doesn't look right in the UI when there is a name-less client